pkgs/by-name/fl/fluent-bit/package.nix

```diff
@@ -72,19 +70,15 @@ stdenv.mkDerivation (finalAttrs: {
     [
       arrow-glib
       c-ares
-      # Needed by rdkafka.
-      curl
       jemalloc
       libbacktrace
-      libnghttp2
+      libnghttp2.dev
```

Comment:
This worked by accident because `curl` pulled in `libnghttp2.dev`

Reply:
`libnghttp2` is alias for `nghttp2.lib`.  

stdenv looks for the `dev` output first but that doesn't exist as `libnghttp2` is an alias for the `lib` output. So we have to be explicit

Reply:
Oh yeh. Good point